draft-ietf-quic-multipath.md

```diff
@@ -381,16 +381,27 @@ with the same, unused Path ID. When the peer receives the PATH_CHALLENGE,
 it MUST pick a Connection ID with the same Path ID for sending the PATH_RESPONSE.
 
 When the multipath extension is negotiated, a client that wants to use an
-additional path MUST first initiate the Address Validation procedure
-with PATH_CHALLENGE and PATH_RESPONSE frames as described in
-{{Section 8.2 of QUIC-TRANSPORT}}, unless it has previously validated
-that address.
+additional path MUST validate the peer's address before sending any data packets
+as described in ({{Section 8.2 of QUIC-TRANSPORT}}),
+unless it has previously validated the four-tuple used for that path.
 
 After receiving packets from the
 client on a new path, if the server decides to use the new path,
-the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
-unless it has previously validated that address.
-An endpoint that receives a PATH_CHALLENGE and does not want to establish
+the server MUST validate the peer's address before sending any data packets
+as described in ({{Section 8.2 of QUIC-TRANSPORT}}),
+unless it has previously validated the four-tuple used for that path.
+Until the client's address is
+validated, the anti-amplification limit from {{Section 8 of QUIC-TRANSPORT}}
+applies.
+
+The server may receive packets for a yet unused Path ID that do not
+contain a path challenge. Such packets are valid if they can be properly decrypted
+and if they contain a valid connection ID.
+As stated above, the server independently needs to perform path validation before sending
+data packets on this new path, and the anti-amplification limit from {{Section 8 of QUIC-TRANSPORT}}
+applies.
```

Comment:
Now we are stating this two points twice in two subsequence paragraphs...? I don't think this sentence is needed.

Reply:
```suggestion
```